app/schemas/models/district.schema.js

```diff
@@ -0,0 +1,13 @@
+const schema = require('./../schemas')
+
+const DistrictSchema = schema.object({}, {
+  countryName: schema.shortString(),
+  ncesId: schema.shortString(),
+  district_schools: schema.int(),
+  district_students: schema.int()
+})
```

Comment:
The kids are upon me so I might add a few other thoughts later on additional properties, but some of the similar comments from school-level additional properties may apply here (website, postalCode, address, urbanity, others...?)